handlers/DeleteEmployee.go

```diff
@@ -2,69 +2,61 @@ package handlers
 
 import (
 	"encoding/json"
+	"errors"
 	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
```

Comment:
just an FYI, we use a tool called `goimports` (in goland, it's in Preferences -> File Watchers) and it usually groups stdlib imports at the top and non-stdlib (github.com) imports at the bottom. We go a step further and group all sendgrid imports in a third group at the bottom. Just a style thing we try to follow.